sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/Framework/Validators/EnumberableItemsNotNullAttributeTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework.Validators;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Tests.Framework.Validators
+{
+    [TestFixture]
+    /// <summary>
+    /// This class will test OneOrMoreRequiredAttribute
+    /// </summary>
+    public class EnumberableItemsNotNullAttributeTests
```

Comment:
```suggestion
    public class EnumerableItemsNotNullAttributeTests
```

Reply:
Updated in [PR](https://github.com/hakimms/azure-sdk-for-net/pull/2)